src/components/FeedActivity.jsx

```diff
@@ -0,0 +1,96 @@
+import { Avatar,  Divider, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+      width: "100%"
+  },
+    avatar: {
+    marginTop: theme.spacing(2),
+    width: theme.spacing(6),
+    height: theme.spacing(6),
+    marginLeft: theme.spacing(3),
+    marginRight: theme.spacing(2)
+  },
+  name: {
+    marginBottom: theme.spacing(1),
+    fontSize: 14
+  },
+  stations: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(2),
+  },
+  date: {
+    marginTop: theme.spacing(0),
+    marginLeft: theme.spacing(3),
```

Comment:
```suggestion
    marginLeft: theme.spacing(2),
```
Dann ist das Datum auf einer Linie mit dem anderen Content.